src/dbus/server/introspect.xml

```diff
@@ -31,6 +31,9 @@
       <arg name="channel_mask" type="u"/>
     </method>
 
+    <method name="Attach">
```

Comment:
missing arguments?

Reply:
It doesn't have one. This is another `Attach` interface that takes no arguments. All the Thread network parameters are passed via the active dataset.